profile.d/toolbox.sh

```diff
@@ -26,8 +26,11 @@ if [ -f /run/ostree-booted ] \
    && [ "${ID}" = "fedora" ] \
    && { [ "${VARIANT_ID}" = "workstation" ] || [ "${VARIANT_ID}" = "silverblue" ] || [ "${VARIANT_ID}" = "kinoite" ]; }; then
     echo ""
-    # shellcheck disable=SC3059
-    echo "Welcome to Fedora ${VARIANT:-$VARIANT_ID}. This terminal is running on the"
+    _pretty='Fedora'
+    if [ -z "${VARIANT}" ] || [ -z "${VARIANT_ID}" ]; then
+      _pretty="${pretty} ${VARIANT_ID:-$VARIANT}"
+    fi
```

Comment:
`${PRETTY_NAME:-Linux}` will make all this unnecessary.  :)

Thanks for taking out the ShellCheck directive to disable [SC3059](https://www.shellcheck.net/wiki/SC3059).

Reply:
NP, exactly why :)